### PR TITLE
Enable reverse rotate

### PR DESCRIPTION
**Pull request by tbaranes**

### CODE REVIEW

File: IBAnimatable/RotationDesignable.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ public protocol RotationDesignable {
 
 public extension RotationDesignable where Self: UIView {
   public func configRotate() {
-    if !rotate.isNaN && rotate > 0 && rotate < 360 {
+    if !rotate.isNaN && rotate > -360 && rotate < 360 {
```
Comment (JakeLin) [inline IBAnimatable/RotationDesignable.swift:18]: Good idea, I will update README.md

**Comment by JakeLin** [comment]
@tbaranes Thanks again and again. if you have time, can you have a look at https://github.com/JakeLin/IBAnimatable/blob/master/Documentation/Roadmap.md ? Any feedback is more than welcome.

**Comment by tbaranes** [comment]
Great documentation, some points I think you could add:
- `Collection View and Collection View Cell` -> + Tableview and TableViewCell
- I won't stroke the slideOut due to the issue #46 

I love this both, that would be really awesome! These both features makes me think a little bit about [iosViews](https://github.com/Charimon/iosViews)

```
Importing Sketch to Swift playground
Importing Sketch to Storyboard? Maybe too hard.
```

I don't have that much idea for others features right now (the MacOS support would be awesome), but I will make a PR if I think about something, if I have sometimes, I will see to make some implementation of some of them :)

Really great library. Thanks for your hard work @JakeLin !

**Comment by JakeLin** [comment]
@tbaranes Thanks for your comments and letting me know about iosViews project, I will look at it to find some ideas. 

The tasks of  "Importing Sketch to Storyboard" are bigger than what iosViews project can do if I understand correctly. Because we need to convert layers into semantic elements e.g. buttons.  There is not one-to-one mapping for flat layer to UIKit elements. And also Sketch doesn't support Auto Layout.  But we maybe able to start with something simple, Sketch can export CSS, or we maybe able to export `IBAnimatable` properties.

**Comment by tbaranes** [comment]
Yes, iosViews is different of IBAnimatable goal (in fact, I never get it working, but it looks great in their README). They are only exporting some bezier path to draw lines... they aren't exporting any `UIKit` elements. As you said, a good start would be to export simple element (a button, a label...) with the right `IBanimatable` configuration.

Is there a place where you detailed more the big features of the roadmap (the small issues are already straightforward)? Some of them have already an issue related (#46 #50). 

By the way, you should add a link to the issue (when an issue is related) in your roadmap, that would be useful in order to check the current state, discussions...

**Comment by JakeLin** [comment]
You are right, I will link the issues to the roadmap and another missing thing is the priority. Next thing I will focus on transitions to develop the architecture to support `IBInspectable` transition animations.

**Pull request merged**